apps/server/swagger/schemas.yml

```diff
@@ -307,37 +310,35 @@ components:
     ChangeHistoryRecord:
       type: object
       properties:
-        comment:
-          type: string
-          description: Comment of the event
         entityName:
           type: string
           description: Name of the Entity
         event:
           type: string
           description: Type of event
-        newData:
+          enum: ['UPDATE', 'DELETE']
+        dataDiff:
           type: object
-          description: New content of the record in JSON format
+          description: Captures the state of the data before the change as `old` and after the change as `new`
```

Comment:
Thinking of Lyric as a "concurrent collaboration" service (multiple users could be updating a document at the same time), `Update` then should be seen as a way to input [operational transformations](https://en.wikipedia.org/wiki/Operational_transformation), meaning the records in Lyric are somewhat equivalent to stateful `deltas`.

This makes me want to advocate for using the word "previous" instead of "old".

(Leave it to me for all your semantic nitpicking needs 😆)

Reply:
I wasn't fully convinced to use `old/new` words, but now that you mention I like "previous" instead of "old", and perhaps "next" instead of "new"?

Reply:
"next" is ambiguous, as it seems to indicate a future state. previous/current makes gramatical sense, but when reading a trail of deltas "current" is also ambiguous. Perhaps previous/updated?

I was OK with old/new in a delta, as it simply describes the two states in the change.

Reply:
`previousState` and `newState` sounds better? or to make it short simply `previous` and `new`?